active/progress/ProgressRing.md

```diff
@@ -69,73 +25,16 @@ The following examples show how to use the IsIndeterminate property to change th
 <ProgressRing IsActive="True" Height="100" Width="100"/>
 ```
 
-![](images/ProgressRing-indeterminate.PNG)
```

Comment:
The Indeterminate ProgressRing image should be updated to reflect the updated designs.

Reply:
This screen shot is from the XAML controls gallery that uses the 2.4 progress ring. Did Albert update the design since then?

Reply:
@karenbtlai I double checked and I think this is the same design. I wonder if the one in XAML controls gallery needs to be updated?

Reply:
The indeterminateProgressRing in XCG the updated one (the solid one with no dots). The image you linked seems to be the dotted one from WUXC?

Reply:
Hm that's weird. I just committed again so maybe it updated that time. It looks correct on my branch in the doc.